terms.html

```diff
@@ -246,6 +246,15 @@
 representation metadata and a potentially unbounded stream of representation
 data." A <a>DID document</a> is a representation of information describing a
 <a>DID subject</a>. See <a href="#representations"></a>.
+  </dd>
+
+    <dt><dfn data-lt="representation-specific entry">representation-specific entries</dfn></dt>
```

Comment:
```suggestion
    <dt><dfn data-lt="representation-specific entry|non-representation-specific entry">representation-specific entries</dfn></dt>
```

I think this needs to be added in order to make that one "non-" reference work?

Reply:
Yes. Applied in 4a345af6b6063f4acc34676000c70df41ef3833c